src/API/Init.php

```diff
@@ -102,8 +81,39 @@ public function rest_api_init() {
 			'Automattic\WooCommerce\Admin\API\NavigationFavorites',
 		);
 
-		// The performance indicators controller must be registered last, after other /stats endpoints have been registered.
-		$controllers[] = 'Automattic\WooCommerce\Admin\API\Reports\PerformanceIndicators\Controller';
+		if ( Features::is_enabled( 'analytics' ) ) {
+			$analytics_controllers = array(
+				'Automattic\WooCommerce\Admin\API\Customers',
+				'Automattic\WooCommerce\Admin\API\Leaderboards',
+				'Automattic\WooCommerce\Admin\API\Reports\Controller',
+				'Automattic\WooCommerce\Admin\API\Reports\Import\Controller',
+				'Automattic\WooCommerce\Admin\API\Reports\Export\Controller',
+				'Automattic\WooCommerce\Admin\API\Reports\Products\Controller',
+				'Automattic\WooCommerce\Admin\API\Reports\Variations\Controller',
+				'Automattic\WooCommerce\Admin\API\Reports\Products\Stats\Controller',
+				'Automattic\WooCommerce\Admin\API\Reports\Variations\Stats\Controller',
+				'Automattic\WooCommerce\Admin\API\Reports\Revenue\Stats\Controller',
+				'Automattic\WooCommerce\Admin\API\Reports\Orders\Controller',
+				'Automattic\WooCommerce\Admin\API\Reports\Orders\Stats\Controller',
+				'Automattic\WooCommerce\Admin\API\Reports\Categories\Controller',
+				'Automattic\WooCommerce\Admin\API\Reports\Taxes\Controller',
+				'Automattic\WooCommerce\Admin\API\Reports\Taxes\Stats\Controller',
+				'Automattic\WooCommerce\Admin\API\Reports\Coupons\Controller',
+				'Automattic\WooCommerce\Admin\API\Reports\Coupons\Stats\Controller',
+				'Automattic\WooCommerce\Admin\API\Reports\Stock\Controller',
+				'Automattic\WooCommerce\Admin\API\Reports\Stock\Stats\Controller',
+				'Automattic\WooCommerce\Admin\API\Reports\Downloads\Controller',
+				'Automattic\WooCommerce\Admin\API\Reports\Downloads\Stats\Controller',
+				'Automattic\WooCommerce\Admin\API\Reports\Customers\Controller',
+				'Automattic\WooCommerce\Admin\API\Reports\Customers\Stats\Controller',
+				'Automattic\WooCommerce\Admin\API\Taxes',
```

Comment:
This isn't reliant on the Reports namespace, so it can be removed from this logic branch (like Products, etc).

Reply:
👍🏽